src/main/java/ro/tripleaunibuc/infrastructure/bootstrap/TreeAProjectApplication.java

```diff
@@ -0,0 +1,13 @@
+package ro.tripleaunibuc.infrastructure.bootstrap;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TreeAProjectApplication {
```

Comment:
Nitpick: we should rename this to TripleAProjectApplication or ThreeAProjectApplication